hearpreprocess/pipeline.py

```diff
@@ -734,19 +734,27 @@ def run(self):
         # minutes or the timestamp embeddings will explode
         sample_duration = self.task_config["sample_duration"]
 
+        # If max_task_duration_by_split is not defined in the task config use the
+        # default MAX_TASK_DURATION_BY_SPLIT
+        max_task_duration_by_split = self.task_config.get(
```

Comment:
You want to copy this otherwise we overwrite the dict.

But it's better to be explicit and create three variables

Reply:
Might also be cleaner to be more explicit here if possible

Reply:
Might also simpler / clarify the if conditions if we do this

Reply:
simplified it with different explicit variables